app/views/sign_up/completions/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<% title @view_model.title %>
+<div class="clearfix col-10 sm-col-12 mx-auto">
+<div class="center">
+  <%= image_tag asset_url(@view_model.image_name), width: 140, alt: '', class: 'mb2'  %>
+</div>
+<h1 class="h3 mb2 my0 sm-mr1 sm-ml1 center"><%= @view_model.heading %></h1>
+<%= render @view_model.service_provider_partial %>
```

Comment:
can we indent this? to make it easier to match `</div>` tags?

Reply:
I presume you mean indent the content for h1.  That's fine.

Reply:
I meant everything inside of `<div class="clearfix` becase we have two `<div>` in a row in the same indentation without a close tag...meaining the second one is inside of the first, right?

Reply:
I see now SMH.  Got it.